api/src/main/java/org/openmrs/api/ConceptService.java

```diff
@@ -173,6 +174,8 @@
 	 *            retire
 	 * @param reason The retire reason
 	 * @return the retired <code>Concept</code> or <code>ConceptNumeric</code>
+	 * @should fail if no reason is given
+	 * @should retire the given concept
 	 * @throws APIException
```

Comment:
@mavrk the @should annotations come last as you've correctly done above. please move them under the throws